src/index.js

```diff
@@ -125,14 +125,24 @@ class GoogleWebfontsPlugin {
         const make = (compilation, cb) => {
 
 			const additionalAssets = (cb) => {
-	            compilation.chunks.add(this.chunk);
+                if (typeof compilation.chunks.add !== 'undefined') {
+                    compilation.chunks.add(this.chunk); // webpack v5
```

Comment:
maybe lets just make this statement a variable called `isWebpack5` - then you don't need the comments.

Reply:
[Here](https://github.com/beyonk-adventures/google-fonts-webpack-plugin/pull/17/commits/10db6e694c0f3ecc7027a81ca614304799935492) you go.

Btw there is quite inconsistency in using tabs/spaces, so I did my best to be coherent with the surroundings.

Reply:
You can install `@beyonk/eslint-config` in another PR if  you like and then it will fit in with our organisation's linting rules :)